docs/internals/contributing.rst

```diff
@@ -46,6 +46,7 @@ Then create a virtual environment and install Batavia into it:
 *For those using anaconda*:
```

Comment:
General review:

- "Setting up dev env" - numbered lists could be useful for skimming/chunking, and it'll spare the usage of "then,", "after that", "lastly", and other redundant text.
- Are the OS-specific steps supposed to be performed after you set up the dev env, and only choose one of them? if so, maybe it's better to group them under a single heading with a purpose stated like "Download and install PhantomJS" (it seems like most of these describe how to do that).